doc/api/dag/l0/public.yml

```diff
@@ -426,6 +426,48 @@ paths:
             Request-Signature:
               $ref: '../../models/headers/Request-Signature.yml'
 
+  /delegated-stake/last-reference/{address}/{tokenLockHash}:
```

Comment:
This should not need 2 parameters, it's sufficient instead to consider a last reference unique across all operations under this (data type, address)
`/delegated-stake/last-reference/{address}`
should be sufficient.